integration/linux/run-tests.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+cd "$(dirname "$0")/../.."
+
+tag_ubuntu_1404=dtbosh-ubuntu14.04
+tag_ubuntu_1604=dtbosh-ubuntu16.04
+project=dtbosh_linux
+docker_compose_opts="--project-name $project --file integration/linux/docker-compose.yml"
+
+echo Starting up services...
+if ! docker-compose $docker_compose_opts up --detach --build ; then
+    echo Task failed
+    exit 1
+fi
+
+function fail {
+    echo Shutting down services...
+    docker-compose $docker_compose_opts down
+
+    echo Task failed
```

Comment:
Quoting of `echo` arguments is missing.
It's working nevertheless, but looks way nicer with quotes :-)